plugins/examples/sidebar/src/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Headlamp, Plugin, Registry } from '@kinvolk/headlamp-plugin/lib';
+import { SectionBox } from '@kinvolk/headlamp-plugin/lib/CommonComponents';
+import Typography from '@material-ui/core/Typography';
+import React from 'react';
+
+class FeedbackSidebar extends Plugin {
+  initialize(registry: Registry) {
+    console.log('feedback initialized');
+
+    registry.registerSidebarItem(null, 'feedback', 'Feedback', '/feedback', {
```

Comment:
Yes. We need to stabilize the API and part of that is converging on an API style for args.
Maybe we deprecate this one and add `registerSidebarEntry` with the same functionality soon? Otherwise I think a deprecation by doing arg matching can be messy.